MonoGame.Framework/Audio/SoundEffect.OpenAL.cs

```diff
@@ -87,8 +85,8 @@ private void PlatformLoadAudioStream(Stream s)
                 afs.ParseBytes (audiodata, false);
                 Size = (int)afs.DataByteCount;
 
-                _data = new byte[afs.DataByteCount];
-                Array.Copy (audiodata, afs.DataOffset, _data, 0, afs.DataByteCount);
+                buffer = new byte[afs.DataByteCount];
```

Comment:
Need to declare buffer before the using statement so it is visibleto the call to BindDataBuffer later.